src/components/container/container.js

```diff
@@ -297,47 +298,53 @@ export default class Container extends UIObject {
   }
 
   playing() {
-    this.trigger(Events.CONTAINER_PLAY, this.name)
+    this.trigger(Events.CONTAINER_PLAY, this.name, this.actionsMetadata.playEvent || {})
   }
 
   paused() {
-    this.trigger(Events.CONTAINER_PAUSE, this.name)
+    this.trigger(Events.CONTAINER_PAUSE, this.name, this.actionsMetadata.pauseEvent || {})
```

Comment:
We should reset the `actionsMetadata` after triggering the event to avoid sending the value more times

Reply:
Done